webapp/src/actions/index.js

```diff
@@ -14,6 +14,14 @@ export const openCreateModal = (postId) => {
     };
 };
 
+export const openCreateModelEmpty = (description, channelId) => (dispatch) => dispatch({
```

Comment:
What do you mean by empty? You are passing in a description and channelId.

Reply:
Good point. d'oh. Fixed. It's a little wordy now, but clear.